MAUI/Calendar/getting-started.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: Getting started with MAUI Calendar control | Syncfusion
+description: Learn here all about getting started with Syncfusion .NET MAUI Calendar(SfCalendar) control and its basic features.
```

Comment:
Learn here all about getting started with Syncfusion .NET MAUI Calendar (SfCalendar) control and its basic features.

Reply:
Description changed